src/auxkernels/TractionAux.C

```diff
@@ -17,45 +17,76 @@
 registerMooseObject("DeerApp", TractionAux);
```

Comment:
Hmm, I saw these changes in another pull request.  Hopefully this doesn't result in problems.